src/modules/media/services/media.service.ts

```diff
@@ -155,11 +189,11 @@ export class MediaService {
 	}
 
 	public async getPlayableUrl(id: string, referer: string): Promise<string> {
-		const embedUrl = trimStart(await this.getEmbedUrl(id), '/');
+		const embedUrl = await this.getEmbedUrl(id);
 
 		const data = {
-			app: 'OR-avo2',
-			client: '', // TODO Required? -- SEE ARC-536
+			app: 'OR-*',
+			client: '', // Asked in ARC-536
```

Comment:
best to leave the "TODO" part of the comment intact, so we can search for all TODO in the code